src/hammer-vlsi/cli_driver_test.py

```diff
@@ -532,5 +533,138 @@ def synthesis_action(self, bad: int) -> dict:  # type: ignore
             BadOverride()
 
 
+class HammerBuildSystemsTest(unittest.TestCase):
+
+    def _read_makefile_lines(self, lines: List[str]) -> Dict[str, List[str]]:
+        targets = {}  # type: Dict[str, List[str]]
+
+        for line in lines:
+            m = re.match(r"^([^.][^\s:]+)\s*:(.*)$", line)
```

Comment:
Can we add a comment here explaining what this regex is searching for?

Reply:
Yep, it's always hard to understand a regex even 5 minutes after writing it...